include/deal.II/grid/reference_cell.h

```diff
@@ -108,6 +109,12 @@ namespace ReferenceCell
     void
     serialize(Archive &archive, const unsigned int /*version*/);
 
+    /**
+     * Return a vector of faces a @p vertex belongs to.
+     */
+    ArrayView<const unsigned int>
+    vertex_faces(unsigned int vertex) const;
```

Comment:
```suggestion
    vertex_faces(const unsigned int vertex) const;
```
Same in the definition.

Reply:
Separately, I'm not in love with the name of the function. Maybe `faces_for_given_vertex()`?